src/sulguk/aiogram_middleware.py

```diff
@@ -115,4 +115,6 @@ def _is_parse_mode_supported(self, method: Any, bot: Bot) -> bool:
         parse_mode = getattr(method, "parse_mode", "")
         if parse_mode is UNSET_PARSE_MODE:
             parse_mode = bot.parse_mode
+        if isinstance(parse_mode, type(UNSET_PARSE_MODE)):
+            parse_mode = bot.default.parse_mode
```

Comment:
This will fail on older versions. We need either specify new version in requirements or find workaround